beacon-chain/core/state/transition.go

```diff
@@ -177,7 +180,7 @@ func ProcessEpoch(state *pb.BeaconState) (*pb.BeaconState, error) {
 
 	// Process receipt roots.
 	if e.CanProcessDepositRoots(state) {
-		e.ProcessDeposits(state)
+		state = e.ProcessEth1Data(state)
```

Comment:
Please add a test for this. With this line removed, no tests would fail (and that is bad)

Reply:
This is still untested